USERS.md

```diff
@@ -80,6 +80,7 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [Walkbase](https://www.walkbase.com/)
 1. [Whitehat Berlin](https://whitehat.berlin) by Guido Maria Serra +Fenaroli
 1. [Yieldlab](https://www.yieldlab.de/)
+1. [Yseop](https://www.yseop.com/)
```

Comment:
Could you please add this change in a different PR ? :)

Reply:
If that can make a difference, I've removed it from this PR.